src/python/pants/backend/python/lint/flake8/subsystem.py

```diff
@@ -8,10 +8,7 @@
 class Flake8(PythonToolBase):
     options_scope = "flake8"
     default_version = "flake8>=3.7.9,<3.8"
-    # `setuptools<45` is for Python 2 support.
-    # TODO: Remove `enum34==1.1.6; python_version == "2.7"` once this enum34 regression is fixed:
-    #  https://bitbucket.org/stoneleaf/enum34/issues/27/enum34-118-broken
-    default_extra_requirements = ["setuptools<45", 'enum34==1.1.6; python_version == "2.7"']
+    default_extra_requirements = ["setuptools<45"]  # NB: `<45` is for Python 2 support
```

Comment:
`enum34` fixed this issue this weekend so we no longer need to pin things. It didn't seem worth it for a dedicated PR and CI cycle, so I snuck it in here.